internal/cmprefimpl/cmpgeos/checks.go

```diff
@@ -702,6 +702,11 @@ var skipIntersection = map[string]bool{
 	"POLYGON((1 0,-0.9 -0.2,-1 -0.0000000000000032310891488651735,-0.9 0.2,1 0))": true,
 	"LINESTRING(0.5 0,0.5000000000000001 0.5)":                                    true,
 	"LINESTRING(0.5 1,0.5000000000000001 0.5)":                                    true,
+
```

Comment:
Because of the way that the reference implementation checks work, new test cases get automatically created for any new geometries that are introduced in unit tests. This is nice because we get a lot of test coverage easily, but because the new geometries introduced in unit tests in this PR cause errors for DECL operations, they need to be added to the skip lists here (otherwise they cause the reference implementation checks to fail).

I don't think this is a very sustainable strategy going forward (there are plans to address this in https://github.com/peterstace/simplefeatures/issues/486).